spec/queries/json_api_spec.rb

```diff
@@ -169,6 +169,47 @@
       end
     end
 
+    context 'when the response error json is in the wrong format' do
+      context 'when the response is a 400 status' do
+        it 'raises an error' do
+          json_api = JsonApi.new(api_root: 'http://hackney.api:8000')
+          stub_request(:get, 'http://hackney.api:8000/properties?postcode=A1%201AA')
+            .to_return(
+              status: 400,
+              body: [{ 'userMessage' => 'That was invalid', 'developerMessage' => 'invalid request' }].to_json
```

Comment:
Is it worth having one of these return multiple items in the array - or did we conclude this would never happen..?

Reply:
Yeah I think it won't, but could in theory. I think it's worth a spec: i'll add one in that commit